src/Events/Integrations/Plugins/Event_Tickets/Emails/Hooks.php

```diff
@@ -48,10 +48,10 @@ public function register(): void {
 	 */
 	protected function add_actions(): void {
 		add_action( 'tribe_template_before_include:tickets/emails/template-parts/body/tickets', [ $this, 'include_event_date_ticket_rsvp_emails' ], 10, 3 );
-		add_action( 'tribe_template_before_include:tickets/emails/template-parts/body/tickets', [ $this, 'include_event_image_ticket_rsvp_emails' ], 10, 3 );
+		add_action( 'tribe_template_before_include:tickets/emails/template-parts/body/tickets', [ $this, 'include_event_image_ticket_rsvp_emails' ], 15, 3 );
 		add_action( 'tribe_template_before_include:tickets/emails/template-parts/header/head/styles', [ $this, 'include_event_ticket_rsvp_styles' ], 10, 3 );
-		add_action( 'tribe_template_after_include:tickets/emails/template-parts/body/tickets', [ $this, 'include_event_venue_ticket_rsvp_emails' ], 10, 3 );
-		add_action( 'tribe_template_after_include:tickets/emails/template-parts/body/tickets', [ $this, 'include_event_calendar_links' ], 10, 3 );
+		add_action( 'tribe_template_after_include:tickets/emails/template-parts/body/post-description', [ $this, 'include_event_venue_ticket_rsvp_emails' ], 15, 3 );
```

Comment:
Setting the same priority will just load them in the order that they're called, we could totally left all in 10, I'm also good with changing the priority to specify the order that way.

Reply:
In case, someone wants to inject something in between, it helps with specified priorities. They can easily inject something with `18` as a priority, otherwise, they have to remove the hook for one and then hook again with a new priority.